omd/packages/maintenance/diskspace

```diff
@@ -188,7 +192,7 @@ def cleanup_aged():
     max_age = time.time() - max_file_age
 
     for plugin in plugins.values():
-        for path, (_size, mtime) in plugin.get('file_infos', {}).items():
+        for path, (_size, mtime) in dict(plugin.get('file_infos', {})).items():
```

Comment:
The more canonical way of handling such a thing is to construct a list of items:
```python
    for path, (_size, mtime) in list(plugin.get('file_infos', {}).items()):
```

We have similar code all over the place...